tests/data_result/module_080_130/models/sale_order.py

```diff
@@ -8,6 +8,12 @@ class SaleOrder(models.Model):
     _inherit = 'sale.order'
 
     custom_field = fields.Char(string="My Custom Field")
+    computed_field = fields.Integer(
+        string="# Computed", compute="_compute_compute_field"
+    )
+
+    def _compute_compute_field(self):
+        self.computed_field = 1
```

Comment:
Part of update about computed field. Should be done in a separate PR

Reply:
Changed in new push